### PR TITLE
feat(buttons): add outline-light appearance (VIV-1775)

### PR DESCRIPTION
**Pull request by rachelbt**

### CODE REVIEW

File: docs/design/appearance.md [hunk 1]
```diff
@@ -532,4 +532,75 @@ currently ghost-light appearance is being used only with `accent` connotation
...
 ```
 
-## Last update: 12.2023
+## Outlined-light
+
+### Used in
+
+`button`, `split-button`
+
+### Important to know
```
Comment (YonatanKra) [inline docs/design/appearance.md:541]: What is important to know? Just the example?

**Review by YonatanKra — CHANGES_REQUESTED** [review]

File: docs/design/appearance.md [hunk 2]
```diff
@@ -532,4 +532,75 @@ currently ghost-light appearance is being used only with `accent` connotation
 </div>
 ```
 
-## Last update: 12.2023
+## Outlined-light
+
+### Used in
+
+`button`, `split-button`
```
Comment (rachelbt) [inline docs/design/appearance.md:539]:
```suggestion
`button`
```

File: docs/design/appearance.md (hunk 1 above)
Comment (rachelbt) [inline docs/design/appearance.md:541]:
```suggestion

```

Comment (rachelbt) [reply docs/design/appearance.md:541]: bad copy paste :)

**Review by YonatanKra — APPROVED** [review]
Can you create a ticket to move the appearance-ui to the docs?

**Comment by rachelbt** [comment]
> Can you create a ticket to move the appearance-ui to the docs?

done: https://jira.vonage.com/browse/VIV-1839

**Pull request merged**